recipe/meta.yaml

```
@@ -16,22 +16,22 @@ source:
     - patches/0015-Don-t-build-redis-with-jemalloc-on-linux-aarch64.patch  # [linux and aarch64]
     - patches/0016-fix-redis-build-on-macos.patch  # [osx]
     - patches/0017-Don-t-use-Xcode-on-macOS.patch
+    - patches/0018-Remove-runtime-agent-dependencies.patch
 
 build:
-  number: 2
-  skip: true  # [py<38 or py>=312]
-  # skipping architectures that do not yet have bazel, so only build for linux64 & win64.
-  # skipping osx and aarch64 for now, as there are complicated bazel errors with external redis (symbols, dependant libraries, and objects).
+  number: 0
+  skip: true  # [py<39 or py>312]
   skip: true  # [(linux and (s390x or ppc64le))]
```

```suggestion
  skip: true  # [linux and s390x]
```
As we do not build for ppc64le anymore

---

@skupr-anaconda Please don't remove things that has no maintenance cost. The services team might want to build for ppc64le.